content/blog/2019/12/2019-12-12-board-election-results.adoc

```diff
@@ -0,0 +1,95 @@
+---
+layout: post
+title: "2019 Jenkins Board and Officer Elections Results"
+tags:
+- community
+- governance
+- governance-board
+- elections
+authors:
+- oleg_nenashev
+opengraph:
+  image: /images/post-images/elections-2019/election-2019-opengraph.png
+---
+
+image:/images/post-images/elections-2019/election-2019-opengraph.png[Jenkins Elections, role=center, float=right, height=178]
+
+The Jenkins community is conducting the 2019 elections for Board and Officer positions.
+The call for nominations has concluded on Nov 25, and the election results have been link:https://groups.google.com/forum/#!topic/jenkinsci-dev/bX2qhzoEWXo[announced in the developer mailing list] on November 28.
+
+On behalf of the Jenkins community, we congratulate all elected board members and officers!
+We also thank Jenkins contributors participated in elections this year: all nominees and hundreds of voters.
+These are the first elections ever conducted by the Jenkins project, and it is a big milestone for Jenkins as a project.
```

Comment:
"These are the first elections ever conducted by the Jenkins project, and it is a big milestone."

Since you reference "the Jenkins project" in the first part of the sentence, you can leave it off at the end.